### PR TITLE
Name column: show only first letter of Realm name

### PR DESCRIPTION
**Pull request by teelolws**
For: https://github.com/kakysha/HonorSpy/issues/179

Also: added sorting alphabetically by Name, and fixed an issue with sorting by Standing sorting randomly if all characters had the same honor last week.

### CODE REVIEW

File: honorspy.lua [hunk 1]
```diff
@@ -133,6 +133,9 @@ local function StartInspecting(unitID)
         if (paused or (not C_PlayerInfo.UnitIsSameServer(PlayerLocation:CreateFromUnit(unitID)))) then
             return
         end
+        
+        -- Disable inspections while in a BG
+        if C_PvP.IsPVPMap() then return end
```
Comment (Slivo-fr) [inline honorspy.lua]: This is a bit of an issue to me, being able to "register" players you meet in BG is kinda big on ERA as the population is quite low and having accurate data is not an easy thing

**Comment by teelolws** [comment]
No I reverted that one, I just couldn't remove it from my fork history without deleting and recreating it. https://github.com/kakysha/HonorSpy/pull/183/commits/6322d57ef90618ccfaa387dae56e7eebd4ab0b6b

**Comment by kakysha** [comment]
Conflicts :( I'm merging in the order of my github notifications :D
Also, why only first letter? No space? Can we actually just make the name occupy the whole width of the column and ellipsis it if it doesn't fit? Truncating player name when there is space left (for short names for example) looks ugly.

**Comment by teelolws** [comment]
Simplest solution. "IlIlIlIlIl" would fit while "WWMWWMWWMW" wouldn't and its too complicated to detect a distinction like that.

**Comment by kakysha** [comment]
Merged this, though I would have made a bit more complex calculation on player names width (at least approx...) so it takes at least the whole column width for short names. Thanks anyway!

**Comment by Slivo-fr** [comment]
> Merged this, though I would have made a bit more complex calculation on player names width (at least approx...) so it takes at least the whole column width for short names. Thanks anyway!

I will take a quick loot at this. Is there any release planned ?

**Pull request merged**